deepcell/image_generators/semantic.py

```diff
@@ -385,3 +399,585 @@ def random_transform(self, x, y=None, seed=None):
 
         self.interpolation_order = _interpolation_order
         return x, y
+
+
+class SemanticMovieIterator(Iterator):
+    """Iterator yielding data from Numpy arrays (X and y).
+
+    Args:
+        train_dict (dict): Dictionary consisting of numpy arrays for X and y.
+        movie_data_generator: Instance of SemanticMovieGenerator
+            to use for random transformations and normalization.
+        batch_size (int): Size of a batch.
+        shuffle (boolean): Whether to shuffle the data between epochs.
+        seed (int): Random seed for data shuffling.
+        data_format (str): One of 'channels_first', 'channels_last'.
+        save_to_dir (str): Optional directory where to save the pictures
+            being yielded, in a viewable format. This is useful
+            for visualizing the random transformations being
+            applied, for debugging purposes.
+        save_prefix (str): Prefix to use for saving sample
+            images (if save_to_dir is set).
+        save_format (str): Format to use for saving sample images
+            (if save_to_dir is set).
+
+    Returns:
+        Iterator yielding data X and y, where y is a list containing
+        transformed masks. Data is randomly augmented using
+        movie_data_generator.
```

Comment:
The "Returns" section is only required for functions that return a value, not a constructor or class definition.

Reply:
```suggestion
```